src/components/felles/minikomponenter.tsx

```diff
@@ -18,3 +18,9 @@ export const Warning = (props: { melding: string }) => (
         <Alert variant="warning">{props.melding}</Alert>
     </div>
 );
+
+export const Info = (props: { melding: string }) => (
+    <div>
+        <Alert variant="info">{props.melding}</Alert>
```

Comment:
Er like greit å bare bruke Alert-komponenten direkte i `cv-innhold.tsx` og `jobbonsker.tsx` fremfor å pakke den inn i en ekstra `<div>`. Hold strukturen så enkel som mulig, det gjør det bedre for skjermlesere blant annet!

Reply:
Fjerna Warning og Info komponenten som tok inn props.
Error melding og Loader e vell greit å ha i egne komponenta? eller tenke du samme om de?

Reply:
Egentlig ja, for det er like greit å bare slenge `midtstill`-klassen rett på designsystem-komponentene, men det er ikke så farlig å fikse akkurat nå!

Reply:
Lagt det til i backloggen!